SYCL/ESIMD/lsc/lsc_usm_load_u64_64.cpp

```diff
@@ -0,0 +1,18 @@
+//==------------ lsc_usm_load_u64_64.cpp - DPC++ ESIMD on-device test
+//---------==//
```

Comment:
Minor comment: Is this line needed? You could fit line1 and 2 into 1 by removing some '-' symbols.